src/components/icon-button/icon-button.scss

```diff
@@ -16,10 +16,6 @@ $height-of-limel-button: functions.pxToRem(36);
     display: none;
 }
 
-:host([disabled]) {
-    pointer-events: none;
-}
-
```

Comment:
First I thought we had this because onClick actions could be set on host (`limel-action-button`) not on the `<button` element within the component. But when testing it seems everything works fine.

https://user-images.githubusercontent.com/35954987/181221405-4e89278e-8305-4ef8-8b77-308e238ff349.mov